src/cpu.c

```diff
@@ -255,11 +255,14 @@ void cpu_step(void) {
 
     // Get and Execute c.opcode
     (*(instr_lookup[cpu.opcode]))();
+
+    // We cannot know (here) the exact number of increments that the PC and cycle count need,
+    // hence the instructions themselves do it
 }
 
 // OP-Codes
 void OPC_NOP(void) {
-    cpu.cycle_count++;
```

Comment:
I know this was my code from the initial reference implementation but do shouldn't the PC also be incremented by 1?

Reply:
Ouh that's a very good catch. Since it has a "length" of 1, I'm quite sure the PC needs to be incremented. I'll see to it!